src/pyvesync/vesyncbasedevice.py

```diff
@@ -1,3 +1,6 @@
+from abc import ABC, ABCMeta
```

Comment:
unneeded import at this time

Reply:
i originally had VeSyncBaseDevice as an ABCMeta but there aren't any abstractmethods so the only reason to have it would be to prevent it from being instantiated.  I wanted to get your opinion on this and test as is first.